diagnostics/etc_composites/util/composites/old/ReadME

```diff
@@ -0,0 +1,19 @@
+CODES
```

Comment:
It looks like the ~/old directory is full of debugging files and backups. Remove them from the PR.